app/src/main/res/layout/mark_stories_completed_story_summary_view.xml

```diff
@@ -33,7 +33,7 @@
       android:paddingBottom="26dp"
       android:text="@{viewModel.storySummary.storyName}"
       android:textAlignment="viewStart"
-      android:textColor="@color/oppia_primary_text"
+      android:textColor="@color/checkbox_text_color"
```

Comment:
This color naming is incorrect.

`component_color_....<meaningful_color_name>_text_color`

Reply:
> This color naming is incorrect.
> 
> `component_color_....<meaningful_color_name>_text_color`

Here `checkbox_text_color` is a xml file having three selection states so how we can name this as `component_color....`

Reply:
Oh okay got it. Lets leave it for now.